pepper-apis/docs/specification/src/endpoints/temporary-users.yml

```diff
@@ -9,5 +9,8 @@ post:
   responses:
     201:
       $ref: '../pepper.yml#/components/responses/CreateTempUserResponse'
+    400:
```

Comment:
Anything in the `responses` map must be a `Response` object, or an object reference- not both. An object reference may not also have a `description`.

Relevant docs:
* [Responses object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#responsesObject)
* [Response object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#responseObject)
* [Reference object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#referenceObject)